src/blob/index/bloom.rs

```diff
@@ -45,32 +45,54 @@ impl Default for Config {
     }
 }
 
+fn m_from_fpr(fpr: f64, k: f64, n: f64) -> f64 {
+    -k * n / (1_f64 - fpr.powf(1_f64 / k)).ln()
+}
+
+fn bits_count_from_formula(config: &Config) -> usize {
+    let max_bit_count = config.max_buf_bits_count; // 1Mb
+    trace!("max bit count: {}", max_bit_count);
+    let k = config.hashers_count;
+    let n = config.elements as f64;
+    trace!("bloom filter for {} elements", n);
+    let mut bits_count = (n * k as f64 / 2_f64.ln()) as usize;
+    let fpr = config.preferred_false_positive_rate;
+    bits_count = bits_count.max(max_bit_count.min(m_from_fpr(fpr, k as f64, n) as usize));
```

Comment:
From config we can setup only `max_bit_count`, so i suggest to prefer its value: 
```
bits_count = max_bit_count.min(bits_count.max(m_from_fpr(fpr, k as f64, n) as usize));
```